README.md

```diff
@@ -1,58 +1,95 @@
-* Fully configurable episode renaming
-* Full integration with SABnzbd and NZBGet
-* Full integration with Kodi, Plex (notification, library update, metadata)
-* Full support for specials and multi-episode releases
-* And a beautiful UI
+[![Installation](https://img.shields.io/badge/wiki-installation-brightgreen.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/Installation)
+[![Docker](https://img.shields.io/badge/wiki-docker-1488C6.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/Installation#Docker)
+[![FAQ](https://img.shields.io/badge/wiki-FAQ-BF55EC.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/FAQ)
+
+- [Install Sonarr for your desired OS](https://github.com/Sonarr/Sonarr/wiki/Installation) or use [Docker](https://github.com/Sonarr/Sonarr/wiki/Installation#Docker)
+- Connect to the UI through http://localhost:8989 or http://your-ip:8989 in your web browser
+- See the [Wiki](https://github.com/Sonarr/Sonarr/wiki) for further configuration
+
+## Downloads
+
+Type | develop (stable) |
+|---|---|
+Binary Releases | [![GitHub Releases](https://img.shields.io/badge/downloads-releases-brightgreen.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/releases)
+Docker (linuxserver.io, x86_64, arm64, armhf) | [![Docker release](https://img.shields.io/badge/linuxserver-sonarr:latest-blue.svg?colorB=1488C6&maxAge=60&style=flat-square)](https://store.docker.com/community/images/linuxserver/sonarr)
+
+
+## Support
+
+[![Donate](https://img.shields.io/badge/donate-bitcoin-gold.svg?maxAge=60&style=flat-square)](https://sonarr.tv/donate)
+
+[![Discord](https://img.shields.io/badge/discord-chat-7289DA.svg?maxAge=60&style=flat-square)](https://discord.gg/M6BvZn5)
+[![Reddit](https://img.shields.io/badge/reddit-discussion-FF4500.svg?maxAge=60&style=flat-square)](https://www.reddit.com/r/sonarr)
+[![GitHub](https://img.shields.io/badge/github-issues-red.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/issues)
+[![GitHub Wiki](https://img.shields.io/badge/github-wiki-181717.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki)
+
+## Status
+
+[![GitHub issues](https://img.shields.io/github/issues/sonarr/sonarr.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/issues)
+[![GitHub pull requests](https://img.shields.io/github/issues-pr/sonarr/sonarr.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/pulls)
+[![Docker Pulls](https://img.shields.io/docker/pulls/linuxserver/sonarr.svg?maxAge=60&style=flat-square)](https://hub.docker.com/r/linuxserver/sonarr)
+[![GNU GPL v3](https://img.shields.io/badge/license-GNU%20GPL%20v3-blue.svg?maxAge=60&style=flat-square)](http://www.gnu.org/licenses/gpl.html)
+[![Copyright 2010-2017](https://img.shields.io/badge/copyright-2019-blue.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr)
```

Comment:
Should be updated to 2019, though in general I’m not a fan of something that has to be updated every year, it’s a waste of everyone’s time.